ChangeLog.md

```diff
@@ -10,6 +10,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 ### Added
 - PSR-4 compliance
 
+## [v1.0.1] - 2023-06-23
+### Security
+- Upgraded signature to match the security standard.
```

Comment:
@bhavya-shukla-lrn @markkellyeire I think we can probably disable this Codacy rule ("Lists should be surrounded by blank lines"), since it goes against our own convention in this file?

Reply:
Will look at this issue of disabling the unwanted rules. Can you flick us the rule set or a confluence page where we can refer the rule set for Codacy